From 01ee5626e2ffa66b2c8c84b70fcc39bebe0ed997 Fri, 15 Jun 2018 11:30:22 +0200
From: Richard Biener <rguenther@suse.de>
Date: Mon, 22 Jan 2018 12:09:07 +0100
Subject: [PATCH] Fix 'debugedit: Couldn't update relocation: invalid data' error for kernel builds. Fixes JB#42171

https://bugzilla.suse.com/show_bug.cgi?id=1076819

diff --git a/tools/debugedit.c b/tools/debugedit.c
index fa47aa5..e0b1d98 100644
--- a/tools/debugedit.c
+++ b/tools/debugedit.c
@@ -2154,9 +2154,9 @@
 		      /* Offset (pointing into the line program) moves
 			 from old to new index including the header
 			 size diff. */
-		      r_offset += ((dso->lines.table[lndx].new_idx
-				    - dso->lines.table[lndx].old_idx)
-				   + dso->lines.table[lndx].size_diff);
+		      r_offset += (ssize_t)((dso->lines.table[lndx].new_idx
+					     - dso->lines.table[lndx].old_idx)
+					    + dso->lines.table[lndx].size_diff);
 
 		      if (rtype == SHT_RELA)
 			{

