From 34e86a556c033fd16273041b5a0d0495edcd1a7b Tue, 29 May 2018 10:27:26 +0200
From: Andrew Branson <andrew.branson@jollamobile.com>
Date: Tue, 29 May 2018 10:26:43 +0200
Subject: [PATCH] Disable systemd_inhibit plugin to minimize dependencies

diff --git a/macros.in b/macros.in
index d086248..d5f1ca6 100644
--- a/macros.in
+++ b/macros.in
@@ -1152,7 +1152,8 @@
 
 # Transaction plugin macros
 %__plugindir		%{_libdir}/rpm-plugins
-%__transaction_systemd_inhibit	%{__plugindir}/systemd_inhibit.so
+# Disabled to minimize dependencies
+#%__transaction_systemd_inhibit	%{__plugindir}/systemd_inhibit.so
 %__transaction_selinux		%{__plugindir}/selinux.so
 %__transaction_syslog		%{__plugindir}/syslog.so
 %__transaction_ima		%{__plugindir}/ima.so
diff --git a/plugins/Makefile.am b/plugins/Makefile.am
index ab4eee3..86965ce 100644
--- a/plugins/Makefile.am
+++ b/plugins/Makefile.am
@@ -22,13 +22,6 @@
 plugins_LTLIBRARIES += selinux.la
 endif
 
-if DBUS
-systemd_inhibit_la_SOURCES = systemd_inhibit.c
-systemd_inhibit_la_CPPFLAGS = $(AM_CPPFLAGS) @DBUS_CFLAGS@
-systemd_inhibit_la_LIBADD = $(top_builddir)/lib/librpm.la $(top_builddir)/rpmio/librpmio.la @DBUS_LIBS@
-plugins_LTLIBRARIES += systemd_inhibit.la
-endif
-
 prioreset_la_SOURCES = prioreset.c
 prioreset_la_LIBADD = $(top_builddir)/lib/librpm.la $(top_builddir)/rpmio/librpmio.la
 plugins_LTLIBRARIES += prioreset.la
