From aee37e98e7b9e4e2e3c8916556b0c19f640ab070 Mon, 4 Jun 2018 16:07:25 +0200
From: Andrew Branson <andrew.branson@jollamobile.com>
Date: Mon, 4 Jun 2018 16:06:59 +0200
Subject: [PATCH] Force libdir to be lib until we've rebuilt libzypp

diff --git a/installplatform b/installplatform
index 12339fc..ede7ad0 100755
--- a/installplatform
+++ b/installplatform
@@ -190,10 +190,6 @@
   # skip architectures for which we dont have full config parameters
   [ -z "$CANONARCH" ] && continue
 
-  if [ "$OS" = "linux" ] && [ "$CANONCOLOR" = 3 ]; then
-      LIB=${LIB}64
-  fi
-
   PPD="${DESTDIR}/${platformdir}/${ARCH}-${OS}"
   [ -d $PPD ] || mkdir -p $PPD
 
